### ISSUE TITLE
Support Fedora 36

### ISSUE THREAD

**Issue by elhananjair** [CloudFire-LLC/cloudfire#759]
I am trying to deploy firezone on Fedora 36 Server, but the minimum supported version by firezone is Fedora 35.

**Comment by jamilbk** [comment]
@elhananjair Does the Fedora 35 package work if you download & install it manually from https://github.com/firezone/firezone/releases ?

**Comment by elhananjair** [comment]
I tried that it get installed fine, but when I try to access the admin panel it does not work, I tried that with my domain. Do I need to install Nginx separately or does the rpm package will install that?

**Comment by elhananjair** [comment]
@jamilbk I didn't have done any tweaks when I installed firezone on Fedora 35, I changed the endpoint and URL, and After that, I could easily access the web UI. But now, the web portal is not showing up, and I have opened the ports on my firewall as I have just done on Fedora 35, I also checked with Apache, and the welcome page showed fine. I have no experience on Ngnix if that's causing the problem.

**Comment by gbe0** [comment]
What happens when you try to access the admin panel? Is there an error?

A few things to check:

* `firezone-ctl status nginx` - Make sure nginx service is running
* `firezone-ctl tail nginx` - Tail the nginx logs. While doing this, do the below requests on the server running Firezone and make sure you see the hits:
 * `curl http://localhost` - Make sure there is a response from web server locally (assuming its on port 80)
 * `curl -k https://localhost.com` - Make sure there is a HTTPS response from web server locally (assuming its on port 443)

If they all look fine, while running `firezone-ctl tail nginx`, try accessing the admin area and see if you can see the requests making it to the server.

**Comment by elhananjair** [comment]
@gbe0 thank you, I think I miss configured firezone configuration which might be the reason. I reinstalled firezone and it worked now. 
Thank you.

**Comment by jamilbk** [comment]
@elhananjair Awesome — I’ll update the install script to detect Fedora 36 and handle accordingly

**Comment by elhananjair** [comment]
Hello @jamilbk, I am having a problem with connecting to the tunnel, specifically with port 51820.
I am using static nat and port forwarded 3001 for accessing the admin panel and port 51820/udp  for wireguard. Now when I check if the port is opened using nmap mypublicip I see only port 3001, port 51280 is not showing up.

**Comment by elhananjair** [comment]
Is there any possibilities that firezone blocks this port?

**Comment by jamilbk** [comment]
@elhananjair We don't block any ingress ports. Does `firezone-ctl tail wireguard` show connection attempts?

**Comment by elhananjair** [comment]
@jamilbk thank you, checked that and this is the output
![image](https://user-images.githubusercontent.com/69236638/176452472-2a55de0a-218b-45a1-9b5c-f8472cd1b98d.png)

I just tried to check if the port is opened using nmap, and it showed closed that's why I was confused. And I wasn't getting handshakes when I checked on the admin panel. But now I retried by removing IPs I specified on Allowed IPs. I am getting bytes send and received:
![image](https://user-images.githubusercontent.com/69236638/176451892-02132df9-f42c-4c36-ab30-ed07f97e0b0b.png)

**Comment by jamilbk** [comment]
What’s the full command you are using for nmap? IIRC you’ll need the -p51820 and -sU flags.

**Comment by elhananjair** [comment]
Yeah I have tried with `nmap -sU 51820 197.156.***.***` and it takes time, `nmap  -sU -p51820 197.156.***.*** ` shows the same results, open and unknown service. even if I change that port number.
The best thing is, now the handshake is being shown, that's great. Currently, I've installed Firezone on a server running Nextcloud, and I added **Public IP** of that server inside Allowed IP lists since I set `www.nextclouddomain.com` to that **Public IP**. I tried to reach out to Nextcloud with that and couldn't access it currently, but other traffic works fine on my phone.
